### PR TITLE
fix: REST type remove default port 6041

### PR DESCRIPTION
**Pull request by huolibo**

### CODE REVIEW

File: src/test/java/com/taosdata/jdbc/cloud/CloudTest.java [hunk 1]
```diff
@@ -17,6 +17,7 @@ public class CloudTest {
     public void connectCloudService() throws Exception {
 
         String url = System.getenv("TDENGINE_CLOUD_URL");
+//        url = "jdbc:TAOS-RS://gw.us-central-1.gcp.cloud.tdengine.com?useSSL=true&token=bb28ba585ffd654f46926ab6897baaa9cf0aaed1";
```
Comment (sunpe) [inline src/test/java/com/taosdata/jdbc/cloud/CloudTest.java:20]: 不应该包含服务器地址和token

File: src/test/java/com/taosdata/jdbc/cloud/ConsumerTest.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+import java.util.Collections;
+import java.util.Properties;
+
+public class ConsumerTest {
+    String url = null;
+
+    @Before
+    public void before() throws SQLException {
+        url = System.getenv("TDENGINE_CLOUD_URL");
+//        url = "jdbc:TAOS-RS://gw.us-central-1.gcp.cloud.tdengine.com?useSSL=true&token=bb28ba585ffd654f46926ab6897baaa9cf0aaed1";
```
Comment (sunpe) [inline src/test/java/com/taosdata/jdbc/cloud/ConsumerTest.java:22]: 代码中不应该写服务器地址和token

File: src/main/java/com/taosdata/jdbc/rs/RestfulStatement.java [hunk 3]
```diff
@@ -121,7 +121,10 @@ private String getUrl() throws SQLException {
             url = protocol + "://" + conn.getHost() + port + "/rest/sql" + dbname + "?token=" + this.conn.getToken() + tz;
         } else {
             String tz = (null == conn.getTz() || "".equals(conn.getTz().trim())) ? "" : "?tz=" + conn.getTz().trim();
-            String port = null != conn.getPort() ? ":" + conn.getPort() : "6041";
+            String port = "";
+            if (null != conn.getPort())
+                port = ":" + conn.getPort();
+
             url = protocol + "://" + conn.getHost() + port + "/rest/sql" + dbname + tz;
         }
```
Comment (sunpe) [inline src/main/java/com/taosdata/jdbc/rs/RestfulStatement.java:129]: 代码可以简化 
String port = null != conn.getPort() ? ":" + conn.getPort() : "";
String tz = (null == conn.getTz() || "".equals(conn.getTz().trim())) ? "" : "&tz=" + conn.getTz().trim();
String url = protocol + "://" + conn.getHost() + port + "/rest/sql" + dbname;

if (this.conn.getToken() != null && !"".equals(this.conn.getToken().trim())) {
    url  = url + "?token=" + this.conn.getToken();
}

url  = url + tz;

**Review by sunpe — APPROVED** [review]
LGTM

**Review by zyyang-taosdata — APPROVED** [review]
LGTM

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**